webapp/channels/package.json

```diff
@@ -224,7 +224,6 @@
     "run": "webpack --progress --watch",
     "dev-server": "webpack serve --mode development",
     "test": "cross-env TZ=Etc/UTC jest --maxWorkers=50%",
-    "test-ci": "cross-env NODE_OPTIONS=--max_old_space_size=5120 TZ=Etc/UTC jest --ci",
```

Comment:
This script is no longer needed since the CI config just calls `npm run test` and passes more options to it